starknet_py/tests/e2e/client/client_test.py

```diff
@@ -204,31 +204,58 @@ async def test_get_class_by_hash(client, class_hash):
 
 
 @pytest.mark.asyncio
-async def test_wait_for_tx_accepted(gateway_client):
+@pytest.mark.parametrize(
+    "client, get_tx_receipt",
+    [
+        (
+            "full_node_client",
+            "starknet_py.net.full_node_client.FullNodeClient.get_transaction_receipt",
+        ),
+        (
+            "gateway_client",
+            "starknet_py.net.gateway_client.GatewayClient.get_transaction_receipt",
+        ),
+    ],
+)
+async def test_wait_for_tx_accepted(client, get_tx_receipt, request):
```

Comment:
One possible problem I'm unsure of. If `client` is already a parametrized fixture, won't this cause this test to run for all client x path combinations e.g. 
GatewayClient and "starknet_py.net.full_node_client.FullNodeClient.get_transaction_receipt"?

cc @drknzz

Reply:
I was not sure about that too. I've checked the CI, and only 2 tests with that name have been ran 👍